### ISSUE TITLE
Error Compiling

### ISSUE THREAD

**Issue by TarekTaha** [unr-arl/rhem_planner#1]
I get this error while trying to compile your code:

> kindr/rotations/RotationEigen.hpp:  No such file or directory

This happened after I installed kindr: https://github.com/ethz-asl/kindr

Does rhem_planner require a specific version of kindr ?

**Comment by papachric** [comment]
Yes it does depend on kindr commit bdca1acddc4e4c0f2e5638d196907e00e996c179 which was the one used with rovio at the time of its snapshot.

With some refactoring the repo now includes this required kindr version as a submodule, and uses catkin build from the catkin_tools pacakge to put it all together (README instructions for this build command are also updated). 

Can you try pulling again and building with catkin build ?

**Comment by TarekTaha** [comment]
I had to install glog-dev to make it compile:
```sudo apt-get install libgoogle-glog-dev```

I tried to launch the package but I get this :

> [rovio-4] restarting process
process[rovio-4]: started with pid [12374]
[rovio-4] process has died [pid 12374, exit code -11, cmd /home/tarek/workspace/rhem_ws/devel/lib/rovio/rovio_node __name:=rovio __log:=/home/tarek/.ros/log/c0c1899c-502d-11e7-bbeb-9cebe837efba/rovio-4.log].

Do I have to modify the launch file ?

**Comment by papachric** [comment]
Yes this is a dependency from other packages. 

The launch file actually includes a number of options that are related with rovio (like camera calibration yaml files) and it's hard to say. 

For a first-time test run you might want to go with the provided demo: https://github.com/unr-arl/rhem_planner/wiki/How-to-use-Demo. This has a prerecorded bag file with visual-inertial data and pointclouds as expeceted, and the respective calibration and configuration files as a good example.

Does this work for you?

**Comment by TarekTaha** [comment]
It compiles, but on two different P.Cs it crashes with the same error : 

> [rovio-6] process has died [pid 20153, exit code -11, cmd /home/tataha/workspace/rhem_planner_ws/devel/lib/rovio/rovio_node __name:=rovio __log:=/home/tataha/.ros/log/faddc316-51e5-11e7-9c08-7c7a913d03c4/rovio-6.log].

it seems that rovio is crashing for some reason. Do you have an idea why ?
